TypeCobol/Compiler/Parser/CodeElementsLinesTokenSource.cs

```diff
@@ -483,6 +486,9 @@ private static IToken BuildCodeElementFromType(CodeElementType type, Token lastT
                     case CodeElementType.EvaluateStatementEnd:
                         codeElement = new EvaluateStatementEnd();
                         break;
+                    case CodeElementType.ExecStatementEnd:
```

Comment:
no case for `CommitStatement` ?

Reply:
Hum, this whole class is dead.
It was used by Antlr for Node parsing.
Usage has been removed by 665e0f6d13048e29dcfbd7b8c22c8e41c90695fc
@mayanje you can delete it.